src/ipahealthcheck/ipa/certs.py

```diff
@@ -879,3 +879,35 @@ def check(self):
             else:
                 yield Result(self, constants.SUCCESS,
                              key=ca)
+
+
+@registry
+class IPAMainCAExpirationCheck(IPAPlugin):
```

Comment:
I see where you're going with the Main but I think I'd drop it and maybe add Chain: IPACAChainExpirationCheck